io-engine/src/core/logical_volume.rs

```diff
@@ -52,3 +54,90 @@ pub struct LvolSpaceUsage {
     /// clone.
     pub allocated_bytes_snapshot_from_clone: Option<u64>,
 }
+
+/// Arguments for replica creation.
+pub struct ReplicaArgs {
```

Comment:
`ReplicaArgs ` -> `CreateReplicaArgs` ?

Reply:
I followed what we do for pool, which is `PoolArgs`. I guess we need to rename that too?